client/dist/styles/carousel.css

```diff
@@ -0,0 +1,64 @@
+/* Wrapper for the entire Image Gallery Component */
+#gallery {
+width: 1000px;
+background-color: grey;
+padding: 50px;
+}
+
+/*The selected image*/
+#slideView img {
+  height: 600px;
+  width: 650px;
+  object-fit: cover;
+  transform: translate(115px);
+  cursor: cell;
+  
+}
+
+.slide{
+  max-height: 500px;
+  max-width: 500px;
+  padding:inherit;
+  }
+
+/*Wrapper for the image carousel*/
+#carousel {
+  width: 860x;
+  height: 220px;
+  background-color: blue;
+  padding: inherit;  
+}
+
+.carousel img {
+    width: 100px;
+    height: 100px;
+    display: block;
+    object-fit: cover;
+    
+}
+
+/* Div of viewable thumbnails. If each thumbnail is 100px + 4 total px of padding = 728px total */
+.thumbView {
```

Comment:
With generic classNames like these, there is a good chance that someone else may have the same className or id.
To be on the safe side, prefix selectors with something that will specify that it is for your component
for example prefixing everything in the QA section with (q- / qa- / a-)